modulefiles/ufs_ursa.intelllvm.lua

```diff
@@ -0,0 +1,30 @@
+help([[
+loads UFS Model prerequisites for Ursa/Intel LLVM
+]])
+
+prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.9.1/envs/ue-oneapi-2024.2.1/install/modulefiles/Core")
+
+stack_oneapi_ver=os.getenv("stack_oneapi_ver") or "2024.2.1"
+load(pathJoin("stack-oneapi", stack_oneapi_ver))
+
+stack_impi_ver=os.getenv("stack_impi_ver") or "2021.13"
+load(pathJoin("stack-intel-oneapi-mpi", stack_impi_ver))
+
+cmake_ver=os.getenv("cmake_ver") or "3.27.9"
+load(pathJoin("cmake", cmake_ver))
+
+load("ufs_common")
+
+nccmp_ver=os.getenv("nccmp_ver") or "1.9.1.0"
+load(pathJoin("nccmp", nccmp_ver))
+
+setenv("CC", "mpiicx")
+setenv("CXX", "mpiicpx")
+setenv("FC", "mpiifx")
+setenv("I_MPI_CC", "icx")
+setenv("I_MPI_CXX", "icpx")
+setenv("I_MPI_FC", "ifx")
```

Comment:
You need to set all three fortran mpi compiler environment variables to ifx or it'll use the ifort compiler. At least, that's what I'm discovering in my project to port UPP to ursa.

```suggestion
setenv("I_MPI_FC", "ifx")
setenv("I_MPI_F90", "ifx")
setenv("I_MPI_F77", "ifx")
```

Reply:
thanks @SamuelTrahanNOAA. will make that change!

Reply:
@ulmononian please verify and mark as resolved, thank you.